dev/com.ibm.websphere.appserver.features/visibility/public/enterpriseBeansRemote-4.0/io.openliberty.enterpriseBeansRemote-4.0.feature

```diff
@@ -3,7 +3,61 @@ symbolicName=io.openliberty.enterpriseBeansRemote-4.0
 visibility=public
 IBM-App-ForceRestart: install, \
  uninstall
-IBM-API-Package: com.ibm.ws.ejb.portable; type="internal"
+IBM-API-Package: com.ibm.ws.ejb.portable; type="internal", \
+  com.ibm.websphere.endpoint; type="ibm-api", \
+  javax.rmi; type="spec", \
```

Comment:
I chose to do that because they are different.  The ejbLite / enterpriseBeansLite has `require-java:="9"` and the ejbRemote / enterpriseBeansRemote does not.

Reply:
Ah yes, I missed that subtle difference.